src/app/shared/ai-bubble/ai-bubble.component.ts

```diff
@@ -23,19 +23,16 @@ import { MatChipsModule } from '@angular/material/chips';
  *
  * @export
  * @class AiBubbleComponent
- * @implements {OnInit}
  */
 @Component({
   selector: 'app-ai-bubble',
   templateUrl: './ai-bubble.component.html',
   standalone: true,
   imports: [MatTooltipModule, MatChipsModule],
 })
-export class AiBubbleComponent implements OnInit {
+export class AiBubbleComponent {
   constructor() {}
 
-  ngOnInit() {}
```

Comment:
This is to satisfy eslint. Updating it caused some linting errors to show up, like this one